statev2/src/storage/db.rs

```diff
@@ -195,12 +287,177 @@ mod test {
     fn test_get_nonexistent() {
         let db = mock_db();
 
-        const TABLE_NAME: &str = "test_table";
         let key_name = "test_key".to_string();
-
         db.create_table(TABLE_NAME).unwrap();
         let val: Option<TestValue> = db.read(TABLE_NAME, &key_name).unwrap();
 
         assert_eq!(val, None);
     }
+
+    /// Tests a read only tx to a table
+    #[test]
+    fn test_ro_tx_simple() {
```

Comment:
thoughts on having a test case for being unable to write in a RO tx?

Reply:
In a `RO` tx the method to write isn't even in scope, so I'm not sure what that would test?